index.md

```diff
@@ -46,3 +51,95 @@ metadata:
 spec:
   minimal: false
 ```
+
+Or, create the resource graphically in the OperatorHub UI:
```

Comment:
Perhaps we should mention that this option is only available if you installed the operator using the bundle image.

Reply:
Good idea, that could be confusing.